CMakeLists.txt

```diff
@@ -216,11 +216,19 @@ include(FindQtRuntime)
 set(CMAKE_AUTOMOC TRUE)
 
 # Find other needed libraries
-find_package(Protobuf REQUIRED)
-if(NOT EXISTS "${Protobuf_PROTOC_EXECUTABLE}")
+find_package(Protobuf CONFIG)
+if(NOT Protobuf_FOUND)
+  find_package(Protobuf REQUIRED)
+endif()
+set(PROTOBUF_LIBRARY "protobuf::libprotobuf")
+set(PROTOBUF_PROTOC_EXECUTABLE "protobuf::protoc")
+
+if(${Protobuf_VERSION} VERSION_LESS "21.0.0" AND NOT EXISTS "${Protobuf_PROTOC_EXECUTABLE}")
```

Comment:
I think it's useful to print where these have been found, see this patch for reference https://gitlab.archlinux.org/archlinux/packaging/packages/cockatrice/-/blob/c2972726ac2ae3fcd52303e9eaee4cee18dbd664/protobuf-23.patch

Reply:
I'm not exactly sure what you're referring to